files/zh-cn/web/javascript/reference/global_objects/number/negative_infinity/index.md

```diff
@@ -3,41 +3,47 @@ title: Number.NEGATIVE_INFINITY
 slug: Web/JavaScript/Reference/Global_Objects/Number/NEGATIVE_INFINITY
 ---
 
-{{JSRef("Global_Objects", "Number")}}
+{{JSRef}}
 
-## 概述
+**`Number.NEGATIVE_INFINITY`** 静态数据属性表示负无穷值。
 
-**`Number.NEGATIVE_INFINITY`** 属性表示负无穷大。
+{{EmbedInteractiveExample("pages/js/number-negative-infinity.html")}}
 
-不用创建一个 {{jsxref("Global_Objects/Number", "Number")}} 实例，使用 `Number.NEGATIVE_INFINITY` 来访问该静态属性。
+## 值
 
-{{js_property_attributes(0,0,0)}}
+与全局属性 {{jsxref("Infinity")}} 的负值相同。
+
+{{js_property_attributes(0, 0, 0)}}
 
 ## 描述
 
-`Number.NEGATIVE_INFINITY` 的值和全局对象的 {{jsxref("Global_Objects/Infinity", "Infinity")}} 属性的负值相同。
+`Number.NEGATIVE_INFINITY` 的表现同数学上的无穷大略有不同：
 
-该值的行为同数学上的无穷大（infinity）有一点儿不同：
+- 任何正值，包括 {{jsxref("Number.POSITIVE_INFINITY", "POSITIVE_INFINITY")}}，乘以 `NEGATIVE_INFINITY` 等于 `NEGATIVE_INFINITY`。
+- 任何负值，包括 `NEGATIVE_INFINITY`，乘以 `POSITIVE_INFINITY` 等于 {{jsxref("Number.POSITIVE_INFINITY", "POSITIVE_INFINITY")}}。
+- 任何正值除以 `NEGATIVE_INFINITY` 都是[负零](https://zh.wikipedia.org/wiki/−0)（正如 [IEEE 754](https://zh.wikipedia.org/wiki/IEEE_754) 中所定义的）。
+- 任何负值除以 `NEGATIVE_INFINITY` 都是[正零](https://zh.wikipedia.org/wiki/0)（正如 [IEEE 754](https://zh.wikipedia.org/wiki/IEEE_754) 中所定义的）。
+- 零除以 `NEGATIVE_INFINITY` 等于 {{jsxref("NaN")}}。
+- {{jsxref("NaN")}} 乘以 `NEGATIVE_INFINITY` 等于 {{jsxref("NaN")}}。
+- `NEGATIVE_INFINITY`，除以任何负值（除了 `NEGATIVE_INFINITY`），都等于 {{jsxref("Number.POSITIVE_INFINITY", "POSITIVE_INFINITY")}}。
+- `POSITIVE_INFINITY`，除以任何正值（除了 {{jsxref("Number.POSITIVE_INFINITY", "POSITIVE_INFINITY")}}），都等于 `NEGATIVE_INFINITY`。
+- `NEGATIVE_INFINITY`，除以 `NEGATIVE_INFINITY` 或 {{jsxref("Number.POSITIVE_INFINITY", "POSITIVE_INFINITY")}}，都等于 {{jsxref("NaN")}}。
+- `x > Number.NEGATIVE_INFINITY` 对于任何不是 `NEGATIVE_INFINITY` 的数字 _x_ 都为真。
 
-- 任何正值，包括 `POSITIVE_INFINITY，`乘以 `NEGATIVE_INFINITY` 为 `NEGATIVE_INFINITY`。
-- 任何负值，包括 `NEGATIVE_INFINITY`，乘以 `NEGATIVE_INFINITY` 为 `POSITIVE_INFINITY`。
-- 0 乘以 `NEGATIVE_INFINITY` 为 `NaN`.
-- NaN 乘以 `NEGATIVE_INFINITY` 为 `NaN`.
-- `NEGATIVE_INFINITY` 除以任何负值（除了 `NEGATIVE_INFINITY）`为 `POSITIVE_INFINITY`。
-- `NEGATIVE_INFINITY` 除以任何正值（除了 `POSITIVE_INFINITY`）为 `NEGATIVE_INFINITY`。
-- `NEGATIVE_INFINITY` 除以 `NEGATIVE_INFINITY` 或 `POSITIVE_INFINITY` 是 `NaN`。
-- 任何数除以 `NEGATIVE_INFINITY` 为 0。
+你可以使用 `Number.NEGATIVE_INFINITY` 属性来表示成功时返回有限数值的错误条件。不过，{{jsxref("NaN")}} 更适合于这种情况。
 
-为了成功返回一个有限值，你可能会使用 `Number.NEGATIVE_INFINITY` 属性来判断是否显示一个条件错误。然而 {{jsxref("Global_Objects/isFinite", "isFinite")}} 方法更适合这种情况。
+因为 `NEGATIVE_INFINITY` 是 {{jsxref("Number")}} 的静态属性，所以你应该始终使用 `Number.NEGATIVE_INFINITY`，而不是一个数字值的属性。
```

Comment:
Could this be aligned with https://github.com/mdn/translated-content/pull/15958/files#diff-b0639b38f99e63839b17bd05159ae23e5d9141f846b6b37e5d9c3c7071e53609R22

Reply:
No problem.